### DISCUSSION TITLE
Different namespaces in request and sample

### DISCUSSION THREAD

**Discussion by AntonUdalow** [swhitley/ERPHelper#98] [General]
Why do you create request with xmlns:bsvc and sample with xmlns:wd???

**Comment by swhitley** [comment]
Hi @AntonUdalow - The samples are Workday-generated and I don't control the prefixes there.  The wd prefix probably originated from RaaS output.  Have you run into any issues running the code with either wd or bsvc?

**Reply by AntonUdalow** [reply]
No, not at all, it's an excellent job that saves me a lot of time, thank you very much!
